scripts/run-smoke-tests.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
Convenience script for running multiple test runner executables based on the comma separated string value of the `AWS_SMOKE_TEST_SERVICE_IDS` environment variable. This will be used in internal build system down the line as well.